appveyor.yml

```diff
@@ -18,18 +18,23 @@ environment:
     - JAVA_HOME: C:\Program Files\Java\jdk1.7.0
   GOOGLE_CLOUD_PROJECT:
     secure: hbX+cAXrU588+iTMKyzGOQ==
-  SECRET:
-    secure: P4H6q90j3t0GC527rKUrpMzYs1z4aqzMh7pfNVqr2ZIJB8H07oeaD0zlosK8An15VQw7q5WTWImiU+yzX1aErct+k2uIah0iArGuEcrKOtQV/4JD10P7QHPsSMXdDXFzbDXsDi6Wf4UDQH7tFGPNU+VrCcXPMfkIlMkYg5tD10TktjdCEJ6YNw0J6SldFeDHDyBkVbmP6SAiQTpqjqO3NDI8i15mH84CWNT90khz4t8Wnhyvnv+J3u89w9Lp/AFoJfkrK2fjDUhR3K1flyG689Skyng2NCSF5wZcq5IPGMRqR+18js4MykeybPVNmEtBBZh+rzPey42oxtQPIX9I6KwPfkXlu2tXuPcbPCpHH5Y= 
+  CREDENTIALS_SECRET:
+    secure: P4H6q90j3t0GC527rKUrpMzYs1z4aqzMh7pfNVqr2ZIJB8H07oeaD0zlosK8An15VQw7q5WTWImiU+yzX1aErct+k2uIah0iArGuEcrKOtQV/4JD10P7QHPsSMXdDXFzbDXsDi6Wf4UDQH7tFGPNU+VrCcXPMfkIlMkYg5tD10TktjdCEJ6YNw0J6SldFeDHDyBkVbmP6SAiQTpqjqO3NDI8i15mH84CWNT90khz4t8Wnhyvnv+J3u89w9Lp/AFoJfkrK2fjDUhR3K1flyG689Skyng2NCSF5wZcq5IPGMRqR+18js4MykeybPVNmEtBBZh+rzPey42oxtQPIX9I6KwPfkXlu2tXuPcbPCpHH5Y=
+  API_KEY_SECRET:
+    secure: tlK82v6dlvXT76+w9PNmdf/oABZbzCdiA7JNLE+iUfQ=
 
 # install required tools (maven, secure-file, encrypted files)
 install:
   - cmd: echo %JAVA_HOME%
   - cmd: echo %M2_HOME%
   - ps: $origin = git config --get remote.origin.url; git clone --quiet -b appveyor $origin appveyor
   - cmd: nuget install secure-file -ExcludeVersion
-  - cmd: IF DEFINED SECRET (secure-file\tools\secure-file -decrypt appveyor\key.json.enc -secret %SECRET% -out appveyor\key.json)
-  - cmd: IF DEFINED SECRET (SET GOOGLE_APPLICATION_CREDENTIALS=%APPVEYOR_BUILD_FOLDER%\appveyor\key.json)
+  - cmd: IF DEFINED CREDENTIALS_SECRET (secure-file\tools\secure-file -decrypt appveyor\key.json.enc -secret %CREDENTIALS_SECRET% -out appveyor\key.json)
+  - cmd: IF DEFINED CREDENTIALS_SECRET (SET GOOGLE_APPLICATION_CREDENTIALS=%APPVEYOR_BUILD_FOLDER%\appveyor\key.json)
   - cmd: ECHO %GOOGLE_APPLICATION_CREDENTIALS%
+  - cmd: IF DEFINED API_KEY_SECRET (secure-file\tools\secure-file -decrypt appveyor\api_key.enc -secret %API_KEY_SECRET% -out appveyor\api_key)
+  - cmd: IF DEFINED API_KEY_SECRET (SET /P GOOGLE_API_KEY=<%APPVEYOR_BUILD_FOLDER%\appveyor\api_key)
+  - cmd: ECHO %GOOGLE_API_KEY%
```

Comment:
Maybe no ECHO here ? since what's echoed is the actual key. Anybody can see this.